### ISSUE TITLE
Markdown diagram : How about Mermaid ?

### ISSUE THREAD

**Issue by GuillaumeDua** [olibre/CodingGoodPractices#1]
Hey Olibre,
What about adding a section about **Mermaid** diagrams for Markdown files.

https://mermaid-js.github.io/mermaid/#/

It's so convinient to create using Visual Code :
[Visual Code : bierner.markdown-mermaid](https://marketplace.visualstudio.com/items?itemName=bierner.markdown-mermaid)

**Comment by olibre** [comment]
Yeah: Great idea :smile: I have already used Mermaid on GitLab CE: https://docs.gitlab.com/ee/user/markdown.html#mermaid  
To be honest, I had started writing a converter Markdown to PDF/HTML/... supporting Mermaid but I lost all my work because I didn't pushed my work in progress and my laptop was stolen. :sob: 

Would you like to pull request your contribution?

What a pity Mermaid is not (yet?) supported by GitHub :disappointed: 

```mermaid
graph TD;
  A-->B;
  A-->C;
  B-->D;
  C-->D;
```
